src/core_read.cpp

```diff
@@ -19,6 +19,7 @@
 #include <boost/algorithm/string/split.hpp>
 
 #include <algorithm>
+#include <limits>
```

Comment:
Is `<limits>` still necessary?

Reply:
Ow, thank you. No it isn't, just removed it.